convene-web/app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,13 @@
+class InvitationsController < ApplicationController
+
```

Comment:
Add authorization to only allow members to invite and remove To Do note since create action exists

Reply:
We did it!